docs/hazmat/backends/interfaces.rst

```diff
@@ -570,6 +570,26 @@ A specific ``backend`` may provide one or more of these interfaces.
         :returns: A new object with the
             :class:`~cryptography.x509.Certificate` interface.
 
+    .. method:: create_x509_crl(builder, private_key, algorithm)
+
+        .. versionadded:: 1.2
+
+        :param builder: An instance of
+            CertificateRevocationListBuilder.
```

Comment:
This will become a link to the class in a future PR. Since the class doesn't exist yet it can't be linked :dancers: